exporter/awsxrayexporter/internal/translator/segment.go

```diff
@@ -172,7 +174,7 @@ func MakeSegment(span ptrace.Span, resource pcommon.Resource, indexedAttrs []str
 		}
 	}
 
-	if name == "" && span.Kind() == ptrace.SpanKindServer {
+	if name == "" && (span.Kind() == ptrace.SpanKindServer || span.Kind() == ptrace.SpanKindConsumer) {
 		// Only for a server span, we can use the resource.
```

Comment:
nit: update the comment as well to reflect the change from only using the resource service name for server spans to using it for server and consumer spans?

Reply:
Changed, thanks.